### ISSUE TITLE
<p> tags & <li> tags have different colors

### ISSUE THREAD

**Issue by samajammin** [ethereum/ethereum-org-website#4495]
**Describe the bug**

`<p>` tags & `<li>` tags have different colors, at least on markdown files.

`<li>`
![Image 2021-11-18 at 3 53 30 PM](https://user-images.githubusercontent.com/8097623/142503273-c496b767-38cb-436f-b739-56947e137266.jpg)

`<p>`
![Image 2021-11-18 at 3 53 02 PM](https://user-images.githubusercontent.com/8097623/142503375-ace2aeae-cbe6-4664-b911-57aedc3d7733.jpg)

This may be specific to the job template? Not sure.

**To Reproduce**

1. Go to https://ethereum.org/en/about/product-designer/
2. Inspect the font color of each

**Expected behavior**

Should be the same font color.

**Screenshots**

See above

**Desktop (please complete the following information):**

All

**Smartphone (please complete the following information):**

All

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by joneskj55** [comment]
Hi @samajammin & @minimalsm I'd like to take this one if it is up for grabs still.

**Comment by samajammin** [comment]
Thanks @joneskj55! Go for it!
